fe/pom.xml

```diff
@@ -161,40 +161,44 @@ under the License.
     </modules>
     <properties>
         <doris.home>${basedir}/../</doris.home>
+        <revision>1.2-SNAPSHOT</revision>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <sonar.organization>apache</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-
         <cglib.version>2.2</cglib.version>
         <commons-cli.version>1.4</commons-cli.version>
+        <commons-filerupload.version>1.5</commons-filerupload.version>
         <commons-codec.version>1.13</commons-codec.version>
         <commons-lang.version>2.6</commons-lang.version>
         <commons-lang3.version>3.9</commons-lang3.version>
         <commons-pool2.version>2.2</commons-pool2.version>
         <commons-validator.version>1.7</commons-validator.version>
-        <gson.version>2.8.6</gson.version>
-        <guava.version>29.0-jre</guava.version>
-        <jackson.version>2.12.3</jackson.version>
+        <gson.version>2.8.9</gson.version>
+        <guava.version>30.0-jre</guava.version>
+        <jackson.version>2.14.2</jackson.version>
         <jackson-mapper-asl.version>1.9.13</jackson-mapper-asl.version>
         <java-cup.version>0.11-a-czt02-cdh</java-cup.version>
         <javassist.version>3.18.2-GA</javassist.version>
         <javax.servlet-api.version>3.0.1</javax.servlet-api.version>
         <je.version>18.3.14-doris-SNAPSHOT</je.version>
         <jflex.version>1.4.3</jflex.version>
         <jmockit.version>1.49</jmockit.version>
-        <commons-io.version>2.6</commons-io.version>
+        <commons-io.version>2.7</commons-io.version>
         <json-simple.version>1.1.1</json-simple.version>
         <junit.version>5.8.2</junit.version>
         <hikaricp.version>3.4.5</hikaricp.version>
         <thrift.version>0.13.0</thrift.version>
+        <tomcat-embed-core.version>8.5.86</tomcat-embed-core.version>
         <log4j2.version>2.18.0</log4j2.version>
+        <log4j-1.2.version>2.18.0</log4j-1.2.version>
+        <slf4j.version>2.0.6</slf4j.version>
         <metrics-core.version>4.0.2</metrics-core.version>
-        <netty-all.version>4.1.42.Final</netty-all.version>
+        <netty-all.version>4.1.89.Final</netty-all.version>
         <objenesis.version>2.1</objenesis.version>
         <grpc.version>1.30.0</grpc.version>
-        <protobuf.version>3.15.0</protobuf.version>
+        <protobuf.version>3.21.12</protobuf.version>
```

Comment:
Is it compatible with 3.15?
The upgrade of rpc framework need to be carefully. Especially thrift and protobuf

Reply:
It is still in testing, but this upgrade is necessary, even if it is not compatible, we need to adapt it. This could be a wide-ranging modification.